samples/brokers/http/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "http"
+version = "0.1.0"
+authors = ["DazWilkin <daz.wilkin@gmail.com>"]
+edition = "2018"
+
+[[bin]]
+name = "standalone"
+path = "src/main.rs"
+
+[[bin]]
+name = "broker"
+path = "src/grpc/broker.rs"
+
+[[bin]]
+name = "client"
+path = "src/grpc/client.rs"
+
+[dependencies]
+akri-shared = { path = "../../../shared" }
```

Comment:
can akri-shared be removed here?

Reply:
Fixed: https://github.com/deislabs/akri/pull/135/commits/975f875dbbcc9243ee6b8740ccc1054aeba8f75f